style7.css

```diff
@@ -0,0 +1,122 @@
+body {
+    max-width: 1510px;
+    font-family:'Raleway', sans-serif ;
+}
+h5 {
+    color: #86878B;
+}
+header {
+    height: 170px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+hr {
+    width: 170px;
+    border: 1px solid #EC7979;
+}
+.mid__one, .mid__two, .mid__four {
+    margin-top: 30px;
+    position: relative;
+    height: 200px;
+    display: flex;
+    justify-content: space-around;
+}
+.mid__one>h5 {
+    width: 370px;
+    margin: 35px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    text-align: center;
+    background: #E2E2E2;
+    color: #86878B;
+    border-radius: 10px;
+}
+.mid__one__h:after {
+    content: "";
+    position: absolute;
+    border: 20px solid transparent; border-top: 30px solid #E2E2E2;
+    margin-top: 180px;
+    margin-right: 200px;
+}
+img {
+    position: absolute;
+    margin-left: -100px;
+}
+.mid__three {
+    height: 440px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    color: #FFF;
+    background-image: url(img/image.png);
```

Comment:
Background picture was not dimmed. As a result this section appears too bright. Here is a few way of how to make it look darker https://www.delftstack.com/howto/css/darken-background-image-css/